src/app/shared/utils/common.ts

```diff
@@ -76,3 +77,29 @@ export function convertArrayToObject<T>(data: T[], keyName: string, valueName: s
     };
   }, {});
 }
+
+export function verifyYAML(data: string): boolean {
+  try {
+    if (!data) {
+      return false;
+    }
+
+    const response = loadYAML(data);
+
+    return typeof response === 'object';
+  } catch (err: unknown) {
```

Comment:
avoid `unknown` type or mention specific reason for that?

any special usage?

Reply:
<img width="760" alt="image" src="https://user-images.githubusercontent.com/56511988/175276310-cbd3ad59-1f54-4d86-a546-2a72592039af.png">
There seems to be an eslint rule requiring this